tools/run-tests.py

```diff
@@ -114,46 +114,49 @@ def create_binary(buildoptions):
 def run_jerry_tests():
     for job in jerry_tests_options:
         ret = create_binary(job.build_args)
+        if ret:
+            break
 
-        if not ret:
-            test_cmd = [TEST_RUNNER_SCRIPT, get_binary_path(job.out_dir), JERRY_TESTS_DIR]
-            if job.test_args:
-                test_cmd.extend(job.test_args)
+        test_cmd = [TEST_RUNNER_SCRIPT, get_binary_path(job.out_dir), JERRY_TESTS_DIR]
+        if job.test_args:
+            test_cmd.extend(job.test_args)
 
-            ret = run_check(test_cmd)
-        else:
+        ret = run_check(test_cmd)
```

Comment:
Why not only use OR for the actual result instead of break at the first failure?